Platform/ElkhartlakeBoardPkg/Library/Stage2BoardInitLib/Stage2BoardInitLib.c

```diff
@@ -1109,15 +1111,9 @@ UpdateFspConfig (
   if (SiCfgData == NULL) {
     DEBUG ((DEBUG_INFO, "Failed to find Silicon CFG!\n"));
   }
-  SecCfgData = (SECURITY_CFG_DATA *)FindConfigDataByTag (CDATA_SECURITY_TAG);
-  if (SecCfgData != NULL) {
-    DEBUG ((DEBUG_INFO, "Load Security Cfg Data\n"));
-    // Configure Sgx SPD Data (Removed due to fsp 2233)
-    //Fspscfg->SgxEpoch0         = SecCfgData->SgxEpoch0;
-    //Fspscfg->SgxEpoch1         = SecCfgData->SgxEpoch1;
-  } else {
-    DEBUG ((DEBUG_INFO, "Failed to find security CFG!\n"));
-  }
+
+  FeaturesCfgData = (FEATURES_CFG_DATA *) FindConfigDataByTag (CDATA_FEATURES_TAG);
+  ASSERT (FeaturesCfgData != NULL);

```

Comment:
A default value to continue to boot if FeaturesCfgData is not found instead of assertion.

Reply:
removed.